ui/webui/resources/BUILD.gn

```diff
@@ -1,12 +1,8 @@
 import("//tools/grit/grit_rule.gni")
 import("//ui/webui/webui_features.gni")
-import("//brave/components/brave_wallet/browser/buildflags/buildflags.gni")
```

Comment:
Mods to this file are no longer needed, as var check is no longer done. Originally added here https://github.com/brave/brave-core/pull/2729/files#diff-48d6a163ef47a669f467774cd3b96e9b and later modified here https://github.com/brave/brave-core/pull/3006/files#diff-48d6a163ef47a669f467774cd3b96e9b